app/controllers/application_controller.rb

```diff
@@ -57,4 +58,14 @@ def logout_user!
   def item_details
     {}
   end
+
+  def set_internal_pages_flash_message
+    return unless Settings.internal_pages_flash_message_html && action_name == 'index'
+
+    Settings.internal_pages_flash_message_config.each do |controller|
```

Comment:
This is all good, but just want to point out that when we want to remove the messages we'll need to set the `internal_pages_flash_message_config` config to an empty array (e.g. not null) since this `.each` assumes an enumerable object.

Reply:
Good catch. Added an empty array to the spec context where nothing is configured.